src/components/AccountSettings.tsx

```diff
@@ -1,330 +1,277 @@
-        history.push(`/authenticated/${val}`)
-    }
+  const handleChangesFn = (val: string) => {
+    history.push(`/authenticated/${val}`)
+  }
+  
+  const markFormDirty = () => setChangeInForm(true)
 
-    // on initial mount, check query parameter for showWelcomeScreen
-    const showWelcomeScreenURLParam = getSearchParam('showWelcomeScreen')
-    if (showWelcomeScreenURLParam && showWelcomeScreenURLParam === 'true' && !isShowingWelcomeScreen) {
-        setIsShowingWelcomeScreen(true)
-    }
-    const onUpdateUserProfile = async (event: React.SyntheticEvent) => {
-        event.preventDefault()
-        try{
-            if(userProfile && editUsername && updatedUsername){
-                userProfile.userName = updatedUsername
-                const updatedProfile = await updateMyUserProfile(userProfile, accessToken)
-                setUserProfile(updatedProfile)
-            }
-            cancelEdit()
-        } catch(err:any) {
-            displayToast(err.reason, 'danger')
-        }
-    }
-
-    useEffect(() => {
-        const getData = async() => {
-            try {
-                const mask =
-                  SynapseConstants.USER_BUNDLE_MASK_ORCID |
-                  SynapseConstants.USER_BUNDLE_MASK_USER_PROFILE |
-                  SynapseConstants.USER_BUNDLE_MASK_IS_VERIFIED |
-                  SynapseConstants.USER_BUNDLE_MASK_IS_CERTIFIED |
-                  SynapseConstants.USER_BUNDLE_MASK_VERIFICATION_SUBMISSION
+  const unpackBundle = (bundle: UserBundle) => {
+    setUserProfile(bundle.userProfile)
+    setUsername(bundle.userProfile?.userName)
+    setFirstName(bundle.userProfile?.firstName)
+    setLastName(bundle.userProfile?.lastName)
+    setPosition(bundle.userProfile?.position)
+    setCompany(bundle.userProfile?.company)
+    setLocation(bundle.userProfile?.location)
+    setIndustry(bundle.userProfile?.industry)
+    setUrl(bundle.userProfile?.url)
+    setVerified(bundle.isVerified)
+    setOrcid(bundle.ORCID)
+    setIsCertified(bundle.isCertified)
+}
 
-                const bundle:UserBundle = await getMyUserBundle(
-                    mask,
-                    accessToken
-                )
-                setUserProfile(bundle.userProfile)
-                setOrcid(bundle.ORCID)
-                setVerfied(bundle.isVerified)
-                setIsCertified(bundle.isCertified)
-                setVerificationSubmission(bundle.verificationSubmission)
-                setVerificationState(bundle.verificationSubmission?.stateHistory?.slice(-1)[0])
+const getUserData = async() => {
+  try{
+    const mask =
+      SynapseConstants.USER_BUNDLE_MASK_ORCID |
+      SynapseConstants.USER_BUNDLE_MASK_USER_PROFILE |
+      SynapseConstants.USER_BUNDLE_MASK_IS_VERIFIED |
+      SynapseConstants.USER_BUNDLE_MASK_IS_CERTIFIED |
+      SynapseConstants.USER_BUNDLE_MASK_VERIFICATION_SUBMISSION  
+    const getPrimaryEmail = await SynapseClient.getNotificationEmail(accessToken)
```

Comment:
You should get the notification emails when asking for your own profile.  Do we need this extra call?

Reply:
The userprofile gives an array of emails which does not specify the notification email